mysql/tests/conftest.py

```diff
@@ -128,12 +128,12 @@ def instance_additional_status():
         'disable_generic_tags': 'true',
         'additional_status': [
             {
-                'name': "innodb_rows_read",
+                'name': "Innodb_rows_read",
```

Comment:
weird looks like these should have been capitalized all along https://github.com/DataDog/integrations-core/blob/master/mysql/datadog_checks/mysql/const.py#L218 ?

Reply:
Yeah I'm trying to figure out why this change is affecting the metrics; that's not clear to me yet, but these values are definitely wrong.